README.md

```diff
@@ -42,15 +42,15 @@ This project provides a convenient distribution of all the code required to get
 
 ## Getting started
 
-Install this library:
+1. Install this library:
 
 ```sh
 npm install @honeycombio/opentelemetry-web @opentelemetry/auto-instrumentations-web
 ```
 
-[Get a Honeycomb API key](https://docs.honeycomb.io/quickstart/#create-a-honeycomb-account).
+2. [Get a Honeycomb API key]([https://docs.honeycomb.io/quickstart/#create-a-honeycomb-account).
```

Comment:
This looks to be the same link, maybe got missed in a commit?

Reply:
Ack! Thank you @JamieDanielson. 🙈 fixing now.

Reply:
Commit added!